src/edgeProxy/edgeProxy.js

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env node
+
+/**
+ * Module dependencies.
+ */
+
+"use strict";
+
+const app = require("./app");
+const debug = require("debug")("edgeproxy:server");
+const http = require("http");
+
+/**
+ * Get port from environment and store in Express.
+ */
+
+const port = normalizePort(process.env.EDGEPROXY_PORT || "4000");
```

Comment:
It's not a part of the core infrastructure but this environment variable should probably have the PDS_ prefix like the others for consistency. Also document in the README